### DISCUSSION TITLE
6월 11일 피어세션

### DISCUSSION THREAD

**Discussion by scarlettJin** [bcaitech1/p4-dkt-lastffang#126] [Peer Session]
브리핑

@ddoyoon:
LGBM 에서는 성능이 없을 수는 있지만,
통계량 계산을 train 기준으로 test 데이터에 적용해주는 작업을 추가 하였고, 리더보드 점수가 제일 높게 나옴.
tag 별로, 그룹별로 해서, test_id 유사한 애들 뽑아서, 작업중.

@scarlettJin:
interaction 형태를 다양한 형태로 시도해봤으나, 성능 향상이 없었음,
시간대와 시간대별 정답률 feature 테스트했으나,성능 향상이 없었음

@alinghi:
userid와 testid 로 그룹해서 돌려보자. 해서 해봤는데, train이 10시간 넘어갔음.
성능도  잘  나오지 않았음.

@gonipark:
밀렸던 강의 안에서 아이디어를 찾을려고 노력중.

@newspring97:
LGBM작업중 사이킷런 버전 문제 해결

@GY-Jeong:
k-fold 와 임베딩 영향 확인 작업 진행했는데, 실제 리더보드에 성능이 좋지 않아서... 

[[Topic]]
knowledge  tag 별 그룹핑 작업 진행중
- 클러스터링 기능 으로 해보자 ( 건희님 진행 예정 ) 
- 비슷한  숫자가 비슷한 종류의 문제지라고 가정함 
- 피어세션-일요일3시

**Comment by newspring97** [comment]
@newspring97
* [ ] LGBM에 Test Dataset의 데이터를 활용하도록 코드 수정 후 테스트해보기
